code/modules/projectiles/updated_projectiles/guns/rifles.dm

```diff
@@ -298,11 +300,11 @@
 /obj/item/weapon/gun/rifle/lmg/set_gun_config_values()
 	fire_delay = config.high_fire_delay
 	burst_amount = config.high_burst_value
-	burst_delay = config.mlow_fire_delay
-	accuracy_mult = config.base_hit_accuracy_mult - config.low_hit_accuracy_mult
+	burst_delay = config.min_fire_delay
+	accuracy_mult = config.base_hit_accuracy_mult
 	accuracy_mult_unwielded = config.base_hit_accuracy_mult - config.max_hit_accuracy_mult
-	scatter = config.med_scatter_value
-	scatter_unwielded = config.max_scatter_value
+	scatter = config.low_scatter_value
```

Comment:
Keep the scatter high. This is a monster weapon.

Reply:
There's a reason it got the wield delay (besides it being thematically appropriate as a bigger, heftier gun.

25% scatter reduction (i.e. 15%, 5% down from 20%) as opposed to a 33% increase in the wield delay.

It's fine.